frontend/src/scenes/feature-flags/FeatureFlagReleaseConditions.tsx

```diff
@@ -516,7 +516,9 @@ export function FeatureFlagReleaseConditions({
                                     <h3 className="l3">Release conditions</h3>
                                     <div className="text-secondary">
                                         Specify {aggregationTargetName} for flag release. Condition sets are evaluated
-                                        from top to bottom. The first condition set that matches will be used.
+                                        from top to bottom. The first condition set that evaluates to true will be used.
+                                        A condition evaluates to true when all its property filters pass AND the target
+                                        falls within the rollout percentage.
```

Comment:
probably overkill, but I'd try and be concise in in-app docs since we want to say as much as possible in as few words.  Maybe try
```
Specify {aggregationTargetName} for flag release. Condition sets are evaluated top to bottom - the first matching set is used. A condition matches when all property filters pass AND the target falls within the rollout percentage.
```
?